packages/core/src/services/health/service.ts

```diff
@@ -663,4 +762,22 @@ export class Service {
 
     return secondsToRecover;
   }
+
```

Comment:
Looks good. The only comment I'd make is if you could group all NEAR related methods together in the file and keep the different chains in alphabetical order. Other than that no issues that I can see.